packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -67,51 +70,97 @@ class AccountImportOptions extends Component {
     }
   };
 
+  handleSignerImported = async ({ address }) => {
```

Comment:
This component is starting to do a lot of different stuff, it would be good to split it out. Not in the scope of this PR though, probably with #264.